app/config/init.js

```diff
@@ -36,6 +36,8 @@ const _init = function (cfg) {
       notify('Error reading configuration: `config` key is missing');
       return _extractDefault(cfg.defaultCfg);
     }
+    // Ignore undefined values in plugin array Issue #1862
+    _cfg.plugins = _cfg.plugins.filter(Boolean);
```

Comment:
Yes, that's right! Will add that. About `localPlugins` I was thinking that it is dev only, so thought it's not needed to run through that check each time unnecessary.